### PR TITLE
Add elevator.js@1.0.0 w/ npm auto-update

### LINKED ISSUE #11269: [Request] Add elevator.js
**Library name:** elevator.js
**Git repository url:** https://github.com/tholman/elevator.js
**npm package name or url** (if there is one): https://www.npmjs.com/package/elevator.js
**License (List them all if it's multiple):** MIT
**Official homepage:** http://tholman.com/elevator.js
**Wanna say something? Leave message here:** This is probably a high priority library.

=====================

Notes from cdnjs maintainer:
Please read the [README.md](https://github.com/cdnjs/cdnjs#cdnjs-library-repository) and [CONTRIBUTING.md](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md) document first.

We encourage you to add a library via sending pull request,
it'll be faster than just opening a request issue,
since there are tons of issues, please wait with patience,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by dakshshah96**
close #11269, cc @tholman

Pull request for issue: #11269

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/tholman/elevator.js
 * Official website (optional, not the repository): http://tholman.com/elevator.js
 * NPM package url (optional): https://www.npmjs.com/package/elevator.js
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 100
   - Count of stars: 4,246
   - Count of forks: 422
 * NPM download stats (optional):
   - Downloads in the last day: 1
   - Downloads in the last week: 10
   - Downloads in the last month: 31

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/elevator.js/package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    "scroll",
+    "top",
+    "elevator"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tholman/elevator.js.git"
+  },
+  "license": "MIT",
+  "author": "Tim Holman <@tholman> (http://tholman.com/)",
```
Comment (sufuf3) [inline ajax/libs/elevator.js/package.json]: I think we can follow the author format as the same with upstream from npm.
```js
  "author": {
    "name": "Tim Holman",
    "email": "timothy.w.holman@gmail.com"
  },
```

File: ajax/libs/elevator.js/package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "elevator.js",
+  "version": "1.0.0",
+  "filename": "elevator.min.js",
+  "description": "Elevator.js fixes those awkward \"scroll to top\" moments the old fashioned way.",
+  "homepage": "http://tholman.com/elevator.js/",
+  "keywords": [
+    "scroll",
+    "top",
+    "elevator"
```
Comment (sufuf3) [inline ajax/libs/elevator.js/package.json]: I think we can follow the author format as the same with upstream from npm.
```js
"keywords": [
  "Elevator",
  "Easter Egg"
], 
```

File: ajax/libs/elevator.js/package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tholman/elevator.js.git"
+  },
```
Comment (sufuf3) [inline ajax/libs/elevator.js/package.json]: I think we can follow the repository format as the same with upstream.
```js
  "repository": {
    "type": "git",
    "url": "git+https://github.com/tholman/elevator.js.git"
  },
```
ref: https://github.com/tholman/elevator.js/blob/master/package.json#L9-L12

File: ajax/libs/elevator.js/package.json (hunk 2 above)
Comment (dakshshah96) [inline ajax/libs/elevator.js/package.json]: @sufuf3 The `Easter Egg` keyword doesn't make any sense.

Also, can we add extra keywords if they are relevant?

Comment (sufuf3) [inline ajax/libs/elevator.js/package.json]: It's good, we can add extra keywords let it better search.
But I still think we can add the same keyword from upstream.

File: ajax/libs/elevator.js/package.json (hunk 3 above)
Comment (dakshshah96) [inline ajax/libs/elevator.js/package.json]: Nor making this change as said by @PeterDaveHello in #11281

**Comment by sufuf3** [comment]
@cdnjs/intern3 
Please help review this PR, Thank you.

**Pull request merged**